configure.ac

```diff
@@ -59,6 +59,20 @@ AS_IF([test $prefix = NONE], [
     prefix=$ac_default_prefix
 ], [])
 
+dnl Fix "exec_prefix" variable if it hasn't been specified...
+AS_IF([test "$exec_prefix" = "NONE"], [
+    AS_IF([test "$prefix" = "/"], [
+	exec_prefix="/usr"
```

Comment:
This isn't correct. If prefix is "/", exec_prefix is also "/" (to construct "/bin" and "/sbin").

Similarly, FHS and GNU say that for prefix==/usr, sharedstatedir==/usr/var (and not /var). So in this case, just set exec_prefix to prefix.